modules/indexing/src/main/java/org/apache/ignite/internal/processors/query/h2/twostep/GridMapQueryExecutor.java

```diff
@@ -668,7 +668,17 @@ private void sendError(ClusterNode node, long qryReqId, Throwable err) {
             GridQueryFailResponse msg = new GridQueryFailResponse(qryReqId, err);
 
             if (node.isLocal()) {
-                U.error(log, "Failed to run map query on local node.", err);
+                if (err instanceof QueryCancelledException) {
+                    String errMsg = "Failed to run cancelled map query on local node: [localNodeId="
+                        + node.id() + ", reqId=" + qryReqId + ']';
+
+                    if (log.isDebugEnabled())
+                        U.warn(log, errMsg, err);
```

Comment:
Cannot realize why do we need to log at WARN level when debug is enabled? From the first glance looks strange.

Reply:
Unfortunately stacktrace can't be printed in info\debug levels.

Reply:
Complicated =( But ok.